tests/test_spectra_utils.py

```diff
@@ -7,83 +7,111 @@
 from simple.utils.spectra import (
     ingest_spectrum,
     # ingest_spectrum_from_fits,
-    spectrum_plottable,
 )
 
 
 @pytest.mark.filterwarnings(
-    "ignore", message=".*Note: astropy.io.fits uses zero-based indexing.*"
+    "ignore",
+    message=".*SAWarning: Column 'Spectra.reference' is marked as a member of the primary key for table 'Spectra'.*",
 )
 @pytest.mark.filterwarnings(
-    "ignore", message=".*'datfix' made the change 'Set MJD-OBS to.*"
+    "ignore", message=".*'kiwi': No known catalog could be found.*"
 )
-@pytest.mark.filterwarnings(
-    "ignore",
-    message=(
-        ".*'erg/cm2/s/A' contains multiple slashes, "
-        "which is discouraged by the FITS standard.*",
-    ),
+@pytest.mark.parametrize(
+    "test_input, message",
+    [
```

Comment:
As a note, if you want to prevent auto-formatters from reformatting text (for example if you prefer something more compact even if it's not Black/ruff/darker-standard) you can preface the block with `# fmt: off` and then end it with `# fmt: on`